stdlib/join/merge_join.go

```diff
@@ -471,7 +471,7 @@ func (r joinRows) getRow(i int, typ semantic.MonoType) values.Object {
 	var obj values.Object
 	for _, chunk := range r {
 		if i > chunk.Len()-1 {
-			i -= chunk.Len() - 1
+			i -= chunk.Len()
```

Comment:
@wolffcm This is the fix for the panic I was talking about. It was causing `getRow` to return a `nil` object, which was then passed into `JoinFn.eval()`, which caused a panic when the compiler tried to call `Retain()` on `nil`.

https://github.com/influxdata/flux/blob/bf17b7dc42455eac4ec7a62c2130f47c0a54695f/stdlib/join/join_fn.go#L124-L125

Reply:
It only triggered when I tried to test `join` with large inputs. Before that, all of the input tables I was using to test it were pretty small, so I imagine there was never a reason for this branch to trigger.